File: python/beautifulMatrix.py
# objective is to have index as 2, 2

matrix = [[int(x) for x in input().strip().split()] for _ in range(5)]

row = col = 0

for i, r in enumerate(matrix):
    if 1 in r:
        row, col = i, r.index(1)
        break

print(abs(row - 2) + abs(col - 2))

File: c++/beautifulMatrix.cpp
/*
Find the index of 1 in the matrix
and find its distance from (2, 2)
*/

#include <iostream>
#include <cmath>
using namespace std;

int main() {
    int n{}, output{};
    for (int i = 0; i < 5; ++i) {
        for (int j = 0; j < 5; ++j) {
            cin >> n;
            if (n == 1) {
                output = abs(i - 2) + abs(j - 2);                
            }
        }
    }
    cout << output << endl;
    return 0;
}
